core/src/main/scala/kafka/metrics/KafkaMetricsConfig.scala

```diff
@@ -29,12 +29,12 @@ class KafkaMetricsConfig(props: VerifiableProperties) {
    * Comma-separated list of reporter types. These classes should be on the
    * classpath and will be instantiated at run-time.
    */
-  val reporters = CoreUtils.parseCsvList(props.getString(KafkaConfig.KafkaMetricsReporterClassesProp,
+  val reporters: Seq[String] = CoreUtils.parseCsvList(props.getString(KafkaConfig.KafkaMetricsReporterClassesProp,
```

Comment:
The Scala 2.13 build failed due to this change. To make it work, you need to add `import scala.collection.Seq`.